community/cloud-foundation/templates/network/examples/network.yaml

```diff
@@ -1,11 +1,11 @@
-# Example on how to use the network templates
+# Example of the network template usage.
 #
-# In this example, a network is created and it shows that subnets could be created
-# via the `subnetworks` property of the `network.py` template (test-subnetwork-1)
+# In this example, a network is created. Subnetworks can be created
+# via the `subnetworks` property of the `network.py` template (such as 
+# test-subnetwork-1)
 #
-# Notice the `name` property for the `templates/network.py` resource type is
-# optional. If omitted in the properties, the DM resource name is used. If 
-# defined in the properties, it takes precedence.
+# Note that the `name` property of the `templates/network.py` resource type is
+# optional. If omitted, the DM resource name is used.
 
 imports:
   - path: templates/network/network.py
```

Comment:
isn't the syntax for creating a sub-network also wrong?

Reply:
The syntax is correct. I deployed this config and its fine.

The subnetwork.py was included in the test config only because it was creating a subnetwork as a separate resource. Notice that test-subnetwork-${RAND}-1 is created within the network.py type. test-subnetwork-${RAND}-2/3 are created as a subnetwork.py type.

Reply:
@alexander-levitski Could you add the following line under the `- path: templates/network/network.py`:

`  name: network.py`

Reply:
Done